### PR TITLE
Update auth0-migration.md

### PR DESCRIPTION
**Pull request by BrJan**
Add clarification that at least one IdP admin must be involved in the migration process.

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `website/static/_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

File: website/docs/docs/cloud/manage-access/auth0-migration.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
----
+ƒƒ---
```
Comment (BrJan) [inline website/docs/docs/cloud/manage-access/auth0-migration.md]: These are typos and should be deleted.

File: website/docs/docs/cloud/manage-access/auth0-migration.md [hunk 2]
```diff
@@ -47,7 +47,7 @@ The fields that will be updated are:
 - Single sign-on URL &mdash; `https://<YOUR_AUTH0_URI>/login/callback?connection={slug}`
 - Audience URI (SP Entity ID) &mdash; `urn:auth0:<YOUR_AUTH0_ENTITYID>:{slug}`
 
-Sample steps to update (you must complete all of them to ensure uninterrupted access to dbt Cloud):
+Sample steps to update (you must complete all of them to ensure uninterrupted access to dbt Cloud and should coordinate making these changes with your identity provider admin):
```
Comment (nghi-ly) [inline website/docs/docs/cloud/manage-access/auth0-migration.md]: looks like important enough information so sugg removing the parenthesis. typically, parenthetical statements describe non-essential information

```suggestion
Below are sample steps to update. You must complete all of them to ensure uninterrupted access to dbt Cloud and you should coordinate with your identity provider admin when making these changes.
```

File: website/docs/docs/cloud/manage-access/auth0-migration.md [hunk 3]
```diff
@@ -69,7 +69,7 @@ Here is an example of an updated SAML 2.0 setup in Okta.
 
 Google Workspace admins updating their SSO APIs with the Auth0 URL won't have to do much if it is an existing setup. This can be done as a new project or by editing an existing SSO setup. No additional scopes are needed since this is migrating from an existing setup. All scopes were defined during the initial configuration. 
 
-Steps to update (you must complete all of them to ensure uninterrupted access to dbt Cloud):
+Steps to update (you must complete all of them to ensure uninterrupted access to dbt Cloud and should coordinate making these changes with your identity provider admin):
```
Comment (nghi-ly) [inline website/docs/docs/cloud/manage-access/auth0-migration.md]:
```suggestion
Below are steps to update. You must complete all of them to ensure uninterrupted access to dbt Cloud and you should coordinate with your identity provider admin when making these changes.
```

File: website/docs/docs/cloud/manage-access/auth0-migration.md [hunk 4]
```diff
@@ -99,7 +99,7 @@ You must complete the domain authorization before you toggle `Enable New SSO Aut
 
 Azure Active Directory admins will need to make a slight adjustment to the existing authentication app in the Azure AD portal. This migration does not require that the entire app be deleted or recreated; you can edit the existing app. Start by opening the Azure portal and navigating to the Active Directory overview.
 
-Steps to update (you must complete all of them to ensure uninterrupted access to dbt Cloud):
+Steps to update (you must complete all of them to ensure uninterrupted access to dbt Cloud and should coordinate making these changes with your identity provider admin):
```
Comment (nghi-ly) [inline website/docs/docs/cloud/manage-access/auth0-migration.md]:
```suggestion
Below are steps to update. You must complete all of them to ensure uninterrupted access to dbt Cloud and you should coordinate with your identity provider admin when making these changes.
```

**Review by nghi-ly — COMMENTED** [review]
tysm for opening this pr, @BrJan ! left one sugg , which is repeated

File: website/docs/docs/cloud/manage-access/auth0-migration.md (hunk 1 above)
Comment (nghi-ly) [inline website/docs/docs/cloud/manage-access/auth0-migration.md]: i think we wanted to delete these 2 characters?

```suggestion
---
```

**Review by nghi-ly — COMMENTED** [review]
looks like the `ƒƒ` is a typo, unless i misunderstood something. thanks!

**Review by nghi-ly — APPROVED** [review]

**Pull request merged**